### ISSUE TITLE
Extension doesn't know about Microsoft.Web/sites/config 2019-08-01

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode-azurearmtools#461]
Extension version: 0.8.3

In the example given on https://docs.microsoft.com/en-us/azure/templates/microsoft.web/2019-08-01/sites/config-appsettings

![image](https://user-images.githubusercontent.com/2193314/73219942-1e318500-4112-11ea-9595-5fda9943b05c.png)

File:

```json
{
  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
  "contentVersion": "1.0.0.0",
  "resources": [
    {
      "name": "appsettings",
      "type": "Microsoft.Web/sites/config",
      "apiVersion": "2019-08-01",
      "properties": {
        "name": "value"
      }
    }
  ]
}
```

Reload Cached Schemas and reload didn't work.

Related: https://github.com/microsoft/vscode-azurearmtools/issues/305, https://github.com/microsoft/vscode-azurearmtools/issues/447

**Comment by neilpeterson** [comment]
@Tyriar thanks. Verified that resource schema has not been generated for this version.

https://github.com/Azure/azure-resource-manager-schemas/tree/master/schemas/2019-08-01

We are working towards getting this generated more quickly / automated.

I've opened an issue in the schemas repository to track. Once a schema has been generated we will suck that into the VS Code ARM tools.

https://github.com/Azure/azure-resource-manager-schemas/issues/847

Thanks
